2020/6/customs_forms.rb

```diff
@@ -0,0 +1,31 @@
+# @param group_answers [Array<String>] the group's answers
+# @return [Integer] the count of unique answers across all group
+# members
```

Comment:
Adding YARD docs for readability and for enhanced type support in VSCode (via vscode-solargraph).

Ex. auto-complete shows `Array` methods:
<img width="353" alt="Screen Shot 2020-12-06 at 11 06 23 AM" src="https://user-images.githubusercontent.com/8866443/101285489-69490980-37b3-11eb-94cc-e1416b287bc2.png">